src/api/product.ts

```diff
@@ -0,0 +1,32 @@
+import { doc, getDoc, collection, getDocs, setDoc, where, query } from 'firebase/firestore';
+import { db } from 'configs/firebase';
+import { TProduct } from 'types';
+
+export const getProduct = async (id: string): Promise<TProduct | null> => {
+  const docRef = doc(db, 'products', id);
```

Comment:
подобные ключи стоит выносить в конфиг (или enum), что-то типа
```ts
const FIREBASE_PATHS = {
  products: 'products',
};
```
```suggestion
  const docRef = doc(db, FIREBASE_PATHS.products, id);
```

Reply:
Понял.